packages/dev/core/src/AudioV2/abstractAudio/subNodes/audioSubNode.ts

```diff
@@ -0,0 +1,5 @@
+export const _AudioSubNode = {
+    STEREO: "Stereo",
+    SPATIAL: "Spatial",
+    VOLUME: "Volume",
+};
```

Comment:
Is union type possible like elsewhere?

Reply:
I'm not sure what you mean. What would be unioned in this case?